Update using-tcpingress.md (#3326)
Kong/docs.konghq.com: app/kubernetes-ingress-controller/2.0.x/guides/using-tcpingress.md

If everything is setup correctly, making a request to Kong should return
HTTP 404 Not Found.

> > **Note**: If you are running the example using Minikube on macos, you may need to run `minikube tunnel` in a separate terminal window.  This will allow external LoadBalancer which is not enabled out of the box.

Reviewer: ```suggestion
> **Note**: If you are running the example using Minikube on MacOS, you may need 
to run [`minikube tunnel`](https://minikube.sigs.k8s.io/docs/handbook/accessing/#loadbalancer-access)
in a separate terminal window.  This exposes LoadBalancer services 
externally, which is not enabled by default.
```
